src/data-model-providers/codegen/CodegenDataModelProvider.h

```diff
@@ -187,6 +187,9 @@ class CodegenDataModelProvider : public DataModel::Provider
 
     void Temporary_ReportAttributeChanged(const AttributePathParams & path) override;
 
+protected:
+    virtual void InitDataModel();
```

Comment:
So is this a temporary thing until we can get the test fixed?  In that case, it might be a good idea to document that, and perhaps name this accordingly.

Reply:
I don't think this is temporary. Since the current unit tests do not involve any cluster implementations, adding the Ember-specific InitDataModelHandler to the unit test framework would introduce unnecessary dependencies.